desktop-ui/program/utility.cpp

```diff
@@ -42,6 +42,8 @@ auto Program::openFile(BrowserDialog& dialog) -> string {
     window.setPath(dialog.path());
     window.setFilters(dialog.filters());
     window.setParent(dialog.alignmentWindow());
+    // Only affects macOS. TODO: are there scenarios where we want to forbid selecting folders?
+    window.setAllowsFolders(true);
```

Comment:
Ideally, we'd only allow selecting folders with extensions matching the given filters, not all folders.

Reply:
I was given the impression that game folders are not required to have an extension?